### ISSUE TITLE
Add a cancel button to a build's page

### ISSUE THREAD

**Issue by half-ogre** [half-ogre/qed#38]
This should only be added after there are users, and only "owner" should be able to do it.

**Comment by forki** [comment]
If it's a public CI the please allow other users (not really admins, but someone you trust) to do the same. 
At teamcity.codebetter.com we see lots of hanging builds and only admins and owners can kill these. That's very unfortunate if they live in another time zone.

This is related to the stuff I wrote in #37.

**Comment by half-ogre** [comment]
@forki: Yeah, user access levels is something I haven't really give any thought to yet, but I think it does make sense to have folks that can manage builds versus manage the server. This is post-0.1, so I probably won't think about it for some time.

**Comment by forki** [comment]
:thumbsup:
